src/component/spinner.js

```diff
@@ -0,0 +1,206 @@
+import React from 'react';
+import {
+  ActivityIndicator,
+  StyleSheet,
+  View,
+  ViewPropTypes,
+} from 'react-native';
+import PropTypes from 'prop-types';
+import { color, font } from './style';
+import Icon from './icon';
+import Text from './text';
+import Svg, { Path, Circle, Defs, Stop, LinearGradient } from './svg';
+
+//
+// Small Spinner
+//
+
+const smallStyles = StyleSheet.create({
+  spinner: {
+    transform: [{ scale: 1.42 }],
+  },
+});
+export const SmallSpinner = ({ ...props }) => (
+  <ActivityIndicator
+    sizeM="small"
```

Comment:
@valentinewallace this prop does not exist. Only `size` https://facebook.github.io/react-native/docs/activityindicator#size